frontend/wallet/src/CallTable.re

```diff
@@ -27,8 +27,14 @@ let nextPending = (t, ~loc) => {
   let task =
     Task.create(cb => {
       incr(t.id);
```

Comment:
I don't know what the execution model is here.  Shouldn't the incr and the deref happen atomically? Naively, given some mild scheduler unfairness, I'd expect to be able to have unboundedly many tasks that end up capturing the same id.